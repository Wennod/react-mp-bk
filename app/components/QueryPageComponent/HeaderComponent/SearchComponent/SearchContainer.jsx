import React from 'react';

import SearchField from './SearchFieldComponent/SearchField';
import SearchButton from './SearchButtonComponent/SearchButton';
import SearchFilter from './SearchFilterComponent/SearchFilter';

class SearchComponentContainer extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        console.log(this.props)
        return (
            <div>
                <SearchField />
                <div className="search-form__controlls">
                    <SearchFilter />
                    <SearchButton getMovies={this.props.getMovie} />
                </div>
            </div>
        );
    }
}

export default SearchComponentContainer;