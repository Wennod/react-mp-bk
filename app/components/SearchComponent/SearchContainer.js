import React from 'react';

import SearchField from './SearchFieldComponent/SearchField';
import SearchButton from './SearchButtonComponent/SearchButton';
import SearchFilter from './SearchFilterComponent/SearchFilter';

class SearchComponentContainer extends React.Component {
    render() {
        return (
            <div>
                <SearchField />
                <div className="search-form__controlls">
                    <SearchFilter />
                    <SearchButton />
                </div>
            </div>
        );
    }
}

export default SearchComponentContainer;