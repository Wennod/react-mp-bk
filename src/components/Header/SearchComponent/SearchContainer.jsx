import React from 'react';

import SearchForm from './SearchFormComponent/SearchForm';

class SearchComponentContainer extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <SearchForm 
                setQueryParam={this.props.setQueryParam} 
                setQueryValue={this.props.setQueryValue} 
                getMovies={this.props.getMovies} 
            />
        );
    }
}

export default SearchComponentContainer;