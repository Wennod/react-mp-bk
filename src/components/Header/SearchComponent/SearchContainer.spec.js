import React from 'react';
import SearchContainer from './SearchContainer';
import renderer from 'react-test-renderer';

describe('SearchContainer snapshot', () => {
    it('should match snapshot', () => {
        const tree = renderer
            .create(<SearchContainer />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});