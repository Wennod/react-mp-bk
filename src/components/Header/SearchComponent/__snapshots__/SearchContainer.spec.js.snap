// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchContainer snapshot should match snapshot 1`] = `
<form
  className="search-form"
  onSubmit={undefined}
>
  <h2>
    Find your movies
  </h2>
  <input
    className="search-form__input"
    id="fieldId"
    name="search"
    onChange={[Function]}
    type="text"
  />
  <div
    className="search-form__group"
  >
    <div
      className="search-form__filter"
    >
      <p
        className="search-form__filter-title"
      >
        Search By
      </p>
      <input
        className="search-form__filter-toggler"
        defaultChecked={true}
        id="title"
        name="searchCriteria"
        onClick={[Function]}
        type="radio"
        value="title"
      />
      <input
        className="search-form__filter-toggler"
        id="genre"
        name="searchCriteria"
        onClick={[Function]}
        type="radio"
        value="genres"
      />
    </div>
    <input
      className="search-button"
      type="submit"
      value="SEARCH"
    />
  </div>
</form>
`;
